src/test/run-pass/allocator-default.rs

```diff
@@ -10,7 +10,7 @@
 
 #![feature(alloc_jemalloc)]
 
-#[cfg(any(target_os = "linux", target_os = "macos"))]
+#[cfg(any(all(target_os = "linux", not(any(target_arch = "aarch64", target_arch = "mips", target_arch = "mips64", target_arch = "powerpc", target_arch = "powerpc64", target_arch = "s390x"))), target_os = "macos"))]
```

Comment:
This seems like it's getting out of hand, perhaps you can just configure the test to only run on x86_64 linux and OSX?